.github/workflows/ecs-deploy-dev.yml

```diff
@@ -5,6 +5,8 @@ on:
   push:
     branches:
       - dev
+    paths-ignore:
+      - 'docs/**'
```

Comment:
Add `localhost` folder as well

Reply:
Approach: List things we know we don't care about (rather than the opposite - in case we add new files we do care about).